Vehicle_Parking_System/Program.cs

```diff
@@ -0,0 +1,41 @@
+using MongoDB.Driver;
+using Vehicle_Parking_Management.Mapping;
+using Vehicle_Parking_Management.Models;
+using Vehicle_Parking_Management.Services;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddSingleton<IVehicleService, VehicleService>();
+builder.Services.AddControllers();
+builder.Services.AddAutoMapper(typeof(AutoMapping));
```

Comment:
Are we using automapper currently?

Reply:
Yes, I am going use to map dto and model object, so I've added to mock the mapper in service class initially.